actionview/test/template/tag_helper_test.rb

```diff
@@ -29,10 +29,11 @@ def test_tag_builder_is_singleton
     assert_equal tag, tag
   end
 
-  def test_tag_options
-    str = tag("p", "class" => "show", :class => "elsewhere")
-    assert_match(/class="show"/, str)
-    assert_match(/class="elsewhere"/, str)
+  def test_tag_builder_options
+    str = tag.p("class" => "show", :class => "elsewhere")
+    assert_match /class="show"/, str
+    assert_match /class="elsewhere"/, str
+    assert_equal "<p class=\"show\" class=\"elsewhere\"></p>", str
```

Comment:
@kaspth I wrapped the arguments in parentheses. thank you @claudiob I removed `assert_equal` thank you